src/components/Footer/Footer.js

```diff
@@ -0,0 +1,56 @@
+import './Footer.css';
+import { MapContainer, TileLayer, Marker, Popup } from 'react-leaflet';
+import messages from '../../assets/local/en';
+import { contacts } from '../../assets/contacts';
+
+const local = messages.footer;
+
+function Footer() {
+  return (
+    <footer className="footer">
+      <div className="footer__info">
+        <div className="footer__contacts">
+          <div className="footer__contact">
+            <strong> {local.address}</strong>: {contacts.address}
```

Comment:
I think this is a good usage of `<strong>`, since the boldness of the text has semantic meaning.